content/en/docs/gke/deploy/upgrade.md

```diff
@@ -24,71 +38,167 @@ However, specific upgrades might need manual actions below.
 
 ## Upgrading management cluster
 
-### From Kubeflow v1.1 to v1.2
+### Upgrade management cluster from v1.1 to v1.2
 
 1. The instructions below assume your current working directory is
 
+   ```bash
+   cd "${MGMT_DIR}"
+   ```
+
+1. Use your management cluster's kubectl context:
+
+   ```bash
+   # Look at all your contexts
+   kubectl config get-contexts
+   # Select your management cluster's context
+   kubectl config use-context "${MGMT_NAME}"
+   # Verify the context connects to the cluster properly
+   kubectl get namespace
+   ```
+
+   If you are using a different enviroment, you can always
+   reconfigure the context by:
+
+   ```bash
+   make create-context
+   ```
+
+1. Check your existing config connector version:
+   ```bash
+   # For Kubeflow v1.1, it should be 1.15.1
+   $ kubectl get namespace cnrm-system -ojsonpath='{.metadata.annotations.cnrm\.cloud\.google\.com\/version}'
+   1.15.1
+   ```
```

Comment:
The suggestion I was going to make doesn't render well. Add a blank line after the end of the code block 😄